roles/matrix-base/tasks/server_base/setup_archlinux.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Install host dependencies
+  pacman:
+    name:
+      - bash-completion
```

Comment:
To be honest, I do not know.
I copied the codes from Debian and CentOS in an intelligent way. It might well be the case that there are unneeded packages within the set of installed packages. Personally I doubt this package is needed somewhere. But I did want to keep things as closely with the other distributions as possible.

Reply:
You're right - i it's not really necessary. I've removed it from everywhere in 26b73e3a4bb.